CI 워크플로우에 gts lint 검사 단계 추가

**Is your feature request related to a problem? Please describe.**

프로젝트는 gts로 코드 스타일을 관리합니다(package.json의 lint/fix 스크립트,
eslint.config.cjs의 gts 설정). lint는 코드의 동작이 아니라 스타일(prettier 포맷 +
일부 eslint 규칙)만 검사하므로, 위반이 있어도 프로그램 동작은 똑같습니다. 즉 버그는
아니지만, 스타일이 어긋난 코드가 쌓이면 코드베이스의 일관성이 무너집니다.

문제는 CI 워크플로우(.github/workflows/bun-cli-check.yml)가 bun install, typecheck,
test, --help만 돌리고 lint는 돌리지 않는다는 점입니다. 그래서 지금은 스타일 일관성이
"각자 푸시 전에 bun run fix / bun run lint를 잊지 않고 돌린다"는 사람의 기억에만
의존합니다. 한 명이라도 깜빡하면 어긋난 코드가 검출 없이 병합됩니다.

실제로 아래 버전 기준 main에서 bun run lint를 돌리면 index.ts에서 16개 오류가
납니다. 누군가 fix를 돌리지 않고 올린 상태가 그대로 병합돼 있다는 뜻입니다.

```
$ bun run lint
$ gts lint
version: 20
/workspaces/reposcore-ts/index.ts
  222:44  error  Insert `⏎··········`  prettier/prettier
  223:1   error  Insert `··`  prettier/prettier
  224:1   error  Insert `··`  prettier/prettier
  225:13  error  Insert `··`  prettier/prettier
  226:1   error  Insert `··`  prettier/prettier
  227:13  error  Insert `··`  prettier/prettier
  228:1   error  Insert `··`  prettier/prettier
  229:11  error  Insert `··`  prettier/prettier
  230:9   error  Replace `}` with `··},⏎········`  prettier/prettier
  241:24  error  Replace `·result.reason·instanceof·Error·?·result.reason.message` with `⏎··············result.reason·instanceof·Error⏎················?·result.reason.message⏎···············`  prettier/prettier
  242:27  error  Replace ``오류:·'${repoPath}'의·선점·현황을·조회할·수·없습니다.·(${msg})`` with `⏎··············`오류:·'${repoPath}'의·선점·현황을·조회할·수·없습니다.·(${msg})`,⏎············`  prettier/prettier
  264:60  error  Replace `detailed,·owner,·repoName` with `⏎··········detailed,⏎··········owner,⏎··········repoName,⏎········`  prettier/prettier
  297:11  error  'summaries:' is defined but never used  no-unused-labels
  300:27  error  Insert `⏎···········`  prettier/prettier
  301:28  error  Insert `⏎···········`  prettier/prettier
  304:25  error  Replace `·result.reason·instanceof·Error·?·result.reason.message` with `⏎············result.reason·instanceof·Error⏎··············?·result.reason.message⏎·············`  prettier/prettier
✖ 16 problems (16 errors, 0 warnings)
  16 errors and 0 warnings potentially fixable with the `--fix` option.
```

16개 중 1개(297:11)는 일반 모드 결과 처리부에 잘못 붙은 라벨 summaries:이고, 나머지
15개는 prettier 포맷 위반입니다. 16개 모두 --fix로 정리되는 항목입니다. CI가 lint를
돌렸다면 이 상태는 병합 전에 막혔을 것입니다.

**Specify version (commit id).**

https://github.com/oss2026hnu/reposcore-ts/commit/072a0e5003354c4c65362fb224b7b41547e11069

**Describe the solution you'd like**

bun-cli-check.yml에 gts lint 검사 단계를 추가해, 스타일 일관성이 사람의 기억이 아니라
CI에서 강제되도록 합니다.

- bun install 다음에 bun run lint(= gts lint) 단계를 추가하고, typecheck·test와 같은
  워크플로우에서 함께 돌도록 둡니다.
- 이 단계는 검사만 합니다. 코드를 자동으로 고치지 않습니다. 위반이 있으면 워크플로우가
  실패하고, 개발자는 로컬에서 bun run fix로 정리한 결과를 커밋하면 됩니다. CI는 올라온
  코드가 통과하는지만 확인합니다.
- 현재 main이 위반 상태이므로, bun run fix로 정리한 결과를 같은 PR에 함께 올린 뒤
  lint 단계를 추가합니다. 위 16개는 모두 --fix로 정리되며, summaries: 라벨도 라벨만
  제거되고 repoSummaries.push(...) 동작은 그대로 유지됩니다.

배치 위치는 typecheck 앞뒤 어디든 무방하며, 스타일 검사를 한 워크플로우 안에서 함께
보이게 하는 것이 목적입니다.

**Describe alternatives you've considered**

lint를 별도 워크플로우 파일로 분리하는 방법도 있으나, 현재 typecheck와 test가
bun-cli-check.yml 한 곳에 모여 있어 lint도 같은 파일에 두는 편이 관리가 단순합니다.

**Additional context**

- gts는 전역 설치가 아니라 devDependency이므로 직접 gts 명령으로는 실행되지 않고
  (bash: gts: command not found), bun run lint / bun run fix처럼 스크립트로 실행해야
  로컬 바이너리가 잡힙니다.
- 이 단계는 스타일만 검사합니다. 동작·로직의 정확성은 typecheck와 bun test(이미 CI에
  있음), 그리고 PR 리뷰가 담당하므로 lint가 그 역할을 대신하지는 않습니다.

--- a/.github/workflows/bun-cli-check.yml
+++ b/.github/workflows/bun-cli-check.yml
@@ -15,6 +15,8 @@ jobs:
 
       - run: bun install
 
+      - run: bun run lint
+
       - run: bun run typecheck
 
       - run: bun test

--- a/index.ts
+++ b/index.ts
@@ -219,15 +219,17 @@ cli
 
       // --claims 옵션이 있으면 점수 계산 대신 이슈 선점 현황만 조회합니다.
       if (isClaimsMode) {
-        const claimTasks = parsedRepos.map(async ({repoPath, owner, repoName}) => {
-          return await githubService.getRecentClaimsData(
-            owner,
-            repoName,
-            claimKeywords,
-            repoPath,
-            useCache,
-          );
-        });
+        const claimTasks = parsedRepos.map(
+          async ({repoPath, owner, repoName}) => {
+            return await githubService.getRecentClaimsData(
+              owner,
+              repoName,
+              claimKeywords,
+              repoPath,
+              useCache,
+            );
+          },
+        );
 
         const claimResults = await Promise.allSettled(claimTasks);
         let hasClaimFailure = false;
@@ -238,8 +240,13 @@ cli
             printClaims(result.value);
           } else {
             hasClaimFailure = true;
-            const msg = result.reason instanceof Error ? result.reason.message : String(result.reason);
-            console.error(`오류: '${repoPath}'의 선점 현황을 조회할 수 없습니다. (${msg})`);
+            const msg =
+              result.reason instanceof Error
+                ? result.reason.message
+                : String(result.reason);
+            console.error(
+              `오류: '${repoPath}'의 선점 현황을 조회할 수 없습니다. (${msg})`,
+            );
           }
         });
 
@@ -261,7 +268,11 @@ cli
           {since},
         );
 
-        const repoData = ScoreCalculator.calculateRepoData(detailed, owner, repoName);
+        const repoData = ScoreCalculator.calculateRepoData(
+          detailed,
+          owner,
+          repoName,
+        );
         const repoSummary = summarizeRepo(repoPath, detailed);
 
         const singleUserScores = sortUserScores(
@@ -294,14 +305,21 @@ cli
         if (result.status === 'fulfilled') {
           const {repoData, repoSummary, written} = result.value;
           repoDataList.push(repoData);
-          summaries: repoSummaries.push(repoSummary);
+          repoSummaries.push(repoSummary);
 
           console.log(`[${repoPath}] CSV 저장: ${written.csv}`);
-          if (written.txt) console.log(`[${repoPath}] TXT 저장: ${written.txt}`);
-          if (written.html) console.log(`[${repoPath}] HTML 저장: ${written.html}`);
+          if (written.txt) {
+            console.log(`[${repoPath}] TXT 저장: ${written.txt}`);
+          }
+          if (written.html) {
+            console.log(`[${repoPath}] HTML 저장: ${written.html}`);
+          }
         } else {
           hasFailure = true;
-          const reason = result.reason instanceof Error ? result.reason.message : String(result.reason);
+          const reason =
+            result.reason instanceof Error
+              ? result.reason.message
+              : String(result.reason);
           console.error(`오류: '${repoPath}'의 데이터를 가져올 수 없습니다.`);
           console.error(`상세 원인: ${reason}`);
         }

